v2v/tests/src/v2v_options.py

```
@@ -946,14 +946,28 @@ def snapshot(disk_info):
+                test.error("esx_ipv6_addr parameter is required for esx_ipv6 checkpoint")
+            cmd = re.sub(ipv4_pattern, esx_ipv6_addr, cmd)
+        if checkpoint == 'vpx_with_invalid_esx_ipv6':
+            ipv4_pattern = r'\b(?:\d{1,3}\.){3}\d{1,3}\b'
+            vpx_ipv6_addr = params.get('vpx_ipv6_addr')
+            if not vpx_ipv6_addr:
+                test.error("esx_ipv6_addr parameter is required for esx_ipv6 checkpoint")
+            cmd = re.sub(ipv4_pattern, vpx_ipv6_addr, cmd)
+            cmd = re.sub(r'/\[(.*?)\]/', r'/\1/', cmd)
+            cmd_export_env = 'LIBVIRT_DEBUG=1'
+            cmd = "%s %s" % (cmd_export_env, cmd)
         cmd_result = process.run(cmd, timeout=v2v_timeout, verbose=True,
```

Ruff (0.15.4)

[error] Function call with `shell=True` parameter identified, security issue

(S604)